docs/zz_generated.kubermaticConfiguration.ce.yaml

```diff
@@ -462,7 +462,7 @@ spec:
   # Versions configures the available and default Kubernetes versions and updates.
   versions:
     # Default is the default version to offer users.
-    default: v1.31.1
+    default: v1.32.3
```

Comment:
Default should always be one the maxSupportedMinor-1 (i.e. 1.31.x), just purely based on convention.